specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -138,6 +140,16 @@ supports customers that want to:
 1. Use the shared component libraries in their projects.
 2. Build their own docker image for data-pipelines
 
+#### Docker
+A Docker image will be published on docker hub under the user https://hub.docker.com/u/versatiledatakit.
+It is built by the CI on any changes to the frontend codebase.
+
+#### Helm(or install helpers)
+There is no use case for having just the frontend installed.
```

Comment:
Is there? And an use case where only the backend is installed?

Reply:
I think there is an argument for only installing the backend 
1. if customers wanted to use their own UI (seems very unlikely)
2. if customers wanted to use just the cli tools, vdk-cli,curl etc... 



We can make the frontend ui configurable similar to how cockroach db is configurable.